packages/kit/src/exports/public.d.ts

```diff
@@ -408,10 +454,29 @@ export interface KitConfig {
 		 */
 		privatePrefix?: string;
 	};
-	/**
-	 * Experimental features which are exempt from semantic versioning. These features may be changed or removed at any time.
-	 */
+	/** Experimental features. Here be dragons. Breaking changes may occur in minor releases. */
```

Comment:
```suggestion
	/** Experimental features. Here be dragons. These are not subject to semantic versioning, so breaking changes or removal can happen in any release.  */
```

Like the ref to "here be dragons" and explicit mentioning of breaking changes. Updating the policy to "only in minors" should happen in a separate PR though, this doesn't only affect tracing but other features under experimental as well.